### DISCUSSION TITLE
how is pg.matrix generated?

### DISCUSSION THREAD

**Discussion by JM-Bader** [vdemichev/DiaNN#498] [Q&A]
So far, I thought that the pg.matrix file is equivalent to the main report tsv filtered for a Global.PG.Q.Value < 1%. 

However, in one example it cought my attention that the pg.matrix file has 25% more protein group entries than the main report tsv after filtering for a Global.PG.Q.Value < 1%.
I then checked the protein groups which were only contained in the pg.matrix table. These all had a Global.PG.Q.Value > 1%: 

![image](https://user-images.githubusercontent.com/98888664/190443549-7ae8a8c6-109b-4b48-aa9c-be04606443fe.png)

Q1: How is the pg.matrix table generated from the main report tsv?
Q2: I thought one should filter the main report tsv for Global.PG.Q.Value < 1% (unless it's a re-search using a DIA-NN library) to get quantitative data to use for biological analyses... Should one ? 
Q3: If Q2 is true, people shouldn't use the pg.matrix table, should they?

**Comment by vdemichev** [comment]
Please see the specification of how matrices are generated in the 'Output' section and how MBR results are filtered (with 'Lib.' q-values) in the 'MBR' section of the docs https://github.com/vdemichev/DiaNN.

**Reply by vdemichev** [reply]
Looks like MBR in your case? How does the log look like?
